rclrs/src/node.rs

```diff
@@ -355,6 +361,12 @@ impl Node {
         domain_id
     }
 
+    /// Gets a parameter given the name.
+    /// Returns None if no parameter with the requested name was found.
+    pub fn get_parameter(&self, name: &str) -> Option<ParameterValue> {
+        self._parameter_map.get(name).cloned()
+    }
+
```

Comment:
This is also a more controversial change.
Creating this API without adding any of the other `Parameter` APIs (specifically `declare_parameter`) would effectively mean that the behavior is what we would have with `automatically_declare_parameters_from_overrides` set to `true`, which is not the case in all other client libraries that default to the parameter set to `false`.
A possibility could be to remove this API until we have at least a `declare_parameter`, which would however make the whole `RosTime` section of the PR dead code.

Reply:
I recommend marking this as a `pub(crate)` function for now so that we can use it internally for `use_sim_time` but aren't committing it to the public API.

I expect our next PR will introduce parameters, so this wouldn't be in the public API for long, but it's probably better for it to not be in at all to avoid confusing anyone.